server/base/src/test/java/org/apache/accumulo/server/util/FileUtilTest.java

```diff
@@ -66,13 +66,13 @@ public void createTmpDir() throws IOException {
 
   @Test
   public void testToPathStrings() {
-    Collection<FileRef> c = new java.util.ArrayList<>();
-    FileRef r1 = createMock(FileRef.class);
-    expect(r1.path()).andReturn(new Path("/foo"));
+    Collection<TabletFile> c = new java.util.ArrayList<>();
+    TabletFile r1 = createMock(TabletFile.class);
+    expect(r1.getNormalizedPath()).andReturn("/foo");
     replay(r1);
     c.add(r1);
-    FileRef r2 = createMock(FileRef.class);
-    expect(r2.path()).andReturn(new Path("/bar"));
+    TabletFile r2 = createMock(TabletFile.class);
```

Comment:
This test seems suspicious.  I am wondering if its still needed.  I will look into this later.

Reply:
I think this test is ok, its just making sure that a method in FileUtil calls a very specific method on TabletFile.  I think the method it is testing could be replaced with a one line stream operation.